pkg/utils/utils.go

```diff
@@ -693,3 +694,29 @@ func isSwitchdev(name string) bool {
 
 	return true
 }
+
+// IsKernelLockdownMode returns true when kernel lockdown mode is enabled
+func IsKernelLockdownMode() bool {
+	out, err := RunCommand("cat", "/host/sys/kernel/security/lockdown")
+	glog.V(2).Infof("IsKernelLockdownMode(): %s, %+v", out, err)
+	if err == nil && strings.Contains(out, "[integrity]") {
```

Comment:
as suggested in previous PR:
actually both [integrity] and [confidentiality] are considered lockdown
so better search for anything other than [none]
https://github.com/torvalds/linux/blob/master/security/lockdown/lockdown.c#L42

Reply:
Modified to "integrity or confidentiality"